modules/rest-api/pages/rest-manage-log-collection.adoc

```diff
@@ -0,0 +1,50 @@
+= Managing Log Collection
+:description: pass:q[Couchbase-Server logs can be collected, prior to inspection.]
+:page-topic-type: reference
+
+[abstract]
+{description}
+
+== HTTP method and URI
+
+----
+POST /controller/startLogsCollection
+
+GET /pools/default/tasks
+
+POST /controller/cancelLogsCollection
+----
+
+== Description
+
+The Couchbase Logging facility records important events, and saves the details to log files, on disk.
+By default, on Linux systems, log files are saved to `/opt/couchbase/var/lib/couchbase/logs`; on MacOS, to `/Users/username/Library/Application Support/Couchbase/var/lib/couchbase/logs`; and on Windows, to `C:\Program Files\Couchbase\Server\var\lib\couchbase\logs`.
+On each node within a Couchbase Server-cluster, logging is performed continuously.
+Logged information can be collected either from all or from selected nodes within the cluster.
+
+The `POST /controller/startLogsCollection` method and endpoint start the collection of Couchbase-Server log files.
```

Comment:
Should "start" be "starts" ?

Reply:
The convention I've used, across multiple pages now, is that 'and' governs plural, and 'with' singular.

Reply:
"method 'and' endpoint start"
Isn't this the 'and' case?

Reply:
Yes, so it governs a plural. To derive a singular, I could change it to something like: 'Used with the POST method, the /controller/startLogsCollection endpoint starts the collection...etc'.

Reply:
Ok, got it. Thank you.